### PR TITLE
Introduce Issue Templates

### PR DESCRIPTION
**Pull request by davidcollom**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+
+**Environment (please complete the following information):**
+ - Version-Checker: [e.g. 0.5.0]
+ - Kubernetes: [e.g. v1.18.6]
+ - Grafana: [e.g. v11.0.0]
+- Prometheus: [e.g. 2.53.0]
```
Comment (hawksight) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]:
```suggestion
  - Prometheus: [e.g. 2.53.0]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+3. The expected metric/container should be '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+
```
Comment (hawksight) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: remove duplicate space line (my OCD)

**Review by hawksight — APPROVED** [review]
Couple of very minor nit picks, but otherwise 👍

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (davidcollom) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: I left two because Githubs Editor normally takes that line 🙈

**Pull request merged**